logdb/logdb.go

```diff
@@ -104,143 +105,174 @@ FROM (%v) e
 	LEFT JOIN ref r5 ON e.topic1 = r5.id
 	LEFT JOIN ref r6 ON e.topic2 = r6.id
 	LEFT JOIN ref r7 ON e.topic3 = r7.id
-	LEFT JOIN ref r8 ON e.topic4 = r8.id`
+	LEFT JOIN ref r8 ON e.topic4 = r8.id
+	%v
+	`
 
 	if filter == nil {
-		return db.queryEvents(ctx, fmt.Sprintf(query, "event"))
+		to, err := newSequence(MaxBlockNumber, 0, 0)
+		if err != nil {
+			return nil, err
+		}
+		where := fmt.Sprintf(" WHERE e.seq >= 0 AND e.seq <= %v", to)
+		return db.queryEvents(ctx, fmt.Sprintf(query, where))
 	}
 
 	var (
-		subQuery = "SELECT seq FROM event WHERE 1"
-		args     []any
+		whereOrderLimit = strings.Builder{}
```

Comment:
Suggestion, so we do not use the empty struct and matches with the initialisation style of the below var `args` (there is another occurrence below, so if you are ok with this the other one should be updated too).

```suggestion
		whereOrderLimit strings.Builder
```